index.html

```diff
@@ -6,6 +6,7 @@
 	<meta name="description" content="Computer Engineering Graduate from the University of Waterloo">
 	<meta name="keywords" content="brandon lam, portfolio, software developer">
 	<meta name="author" content="Brandon Lam">
+	<meta name="true-identity" content="clam">

```

Comment:
```suggestion
	<meta name="true-identity" content="bclam">

```
he is the b to the clam